docs/reference/transform/checkpoints.asciidoc

```diff
@@ -55,6 +55,47 @@ TIP: If the cluster experiences unsuitable performance degradation due to the
 {transform}, stop the {transform} and refer to <<transform-performance>>.
 
 
+[discrete]
+[[sync-field]]
+== Using the ingest timestamp for syncing the {transform}
+
+In most cases, it is strongly recommended to use the ingest timestamp of the 
+source indices for syncing the {transform}. This is the most optimal way for 
+{transforms} to be able to identify new changes. It can be set by using an 
+ingest pipeline. You can create an ingest pipeline either using the 
+<<put-pipeline-api, ingest pipeline API>> (like the example below) or via {kib} 
+under **Stack Management > Ingest Pipelines**. Use a `set` processor to set a 
```

Comment:
can "`set` processor" be linked?

Reply:
Sure, added in [f69dc50](https://github.com/elastic/elasticsearch/pull/87650/commits/f69dc506a7c4cba95927b1d1644c8d3b7afda320).